testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -3375,14 +3375,14 @@ QUnit.module('default options', {
     });
 
 
-    QUnit.test('changing popupWidth in default options should change popover width', function(assert) {
+    QUnit.test('changing dropDownOptions.width in default options should change popover width', function(assert) {
         const defaultWidth = 100;
 
         Lookup.defaultOptions({
             options: {
                 usePopover: true,
                 fullScreen: false,
-                popupWidth: defaultWidth
+                'dropDownOptions.width': defaultWidth
```

Comment:
What happens if a user changes a default value `popupWidth` via the `defaultOptions` method?

Reply:
fixed